[WCMSFEQ-1322]  DCEG video player padding

mltnewman opened the pull request:
[WCMSFEQ-1322] [Change Request] DCEG Video Player Padding
Inline video player on DCEG (set to left or right) had a forced zero-set left margin, so content was flush against the video player.  Created two video classes (video-lft and video-rt) to adopt the same 41.75px margins currently implemented on cancergov.  Removed the zero left margin from all three existing video size classes (.size50, .size75, and .size100).

alifr commented:
I don't know that I like these new classes. This mean that a figure would have to have multiple classes to display properly. 
For example `<figure class="video-rt right size50">`.  `right` is needed to float the element to the right and the `video-rt` class is needed to apply the necessary margin to the right. Why isn't it just `.video.right {margin-left: 41.75px;}` Was it necessary to make a new class that has to be applied to the content items? Also, the `.video.right` combo selector does not make assumptions about float position. Then you don't have potential issues like `<figure class="video-rt left size50">` where left and right conflict.

mltnewman commented:
Initially created only the '.video-rt' and '.video-lft' classes with just the margin adjustments, because there was no .video class, and the .videoPlayer class has a very specific margin set.  The .right and .left was need to float an element right or left, and is also in use in several other elements.  I have now updated '.video-rt' and '.video-lft' with the appropriate floats, so that the .right and/or .left class can be dropped from the video alignment classes.  I also added a '.video-cntr' class to center align a video instance.

The implementation would be as follows:
<figure class="video-rt size50"> or <figure class="video-rt size75">
to float a 50% (or 75%) sized video to the *right* with appropriate margins, 

<figure class="video-lft size50"> or <figure class="video-lft size75">
to float a 50% (or 75%) sized video to the *left* with appropriate margins, 

and
<figure class="video-cntr size50"> or <figure class="video-cntr size75">
to center align a 50% (or 75%) sized video.
